superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -512,6 +513,7 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
     const data: any = {
       ...currentAlert,
       type: isReport ? 'Report' : 'Alert',
+      force_screenshot: contentType === 'chart' && !isReport ? 'true' : 'false',
```

Comment:
This is made configurable in https://github.com/apache/superset/pull/17855/files